Assets/Scripts/Models/Temperature.cs

```diff
@@ -1,6 +1,6 @@
 ﻿#region License
 // ====================================================
-// Project Porcupine Copyright(C) 2016 Team Porcupine
+// Project Porcupine Copyright(c) 2016 Team Porcupine
```

Comment:
I would assume so, its a capital c in the other license header. Fun fact: theres a stylecop rule that forces you to use the actual copyright character.